src/main/java/com/conveyal/datatools/manager/jobs/NotifyAdminUsersJob.java

```diff
@@ -0,0 +1,49 @@
+package com.conveyal.datatools.manager.jobs;
+
+import com.conveyal.datatools.manager.models.Project;
+import com.conveyal.datatools.manager.persistence.Persistence;
+import com.conveyal.datatools.manager.utils.JobUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static com.conveyal.datatools.manager.auth.Auth0Users.getVerifiedEmailsForAdmins;
+import static com.conveyal.datatools.manager.utils.NotificationsUtils.sendNotification;
+
+public class NotifyAdminUsersJob extends NotifyUsersJob {
+
+    public static final Logger LOG = LoggerFactory.getLogger(NotifyAdminUsersJob.class);
+    private final String target;
+    private final String message;
```

Comment:
Move `target` and `message` to the parent class (these fields are also found in the subscription job).

Reply:
Addressed in: https://github.com/ibi-group/datatools-server/pull/616/commits/b38e277e13b2565ec67c994491a31a3229a256c7